core/src/main/java/apoc/convert/Convert.java

```diff
@@ -214,7 +214,9 @@ private Long parseLongString(String input) {
         } catch (NumberFormatException nfe) {
             try {
                 return (long)Double.parseDouble(input);
-            } catch(NumberFormatException nfe2) {}
+            } catch(NumberFormatException nfe2) {
+                // String was not able to be parsed, return null
```

Comment:
Maybe do `return null` directly here. As it is, the catch block is still empty as the ticket was saying. Not that I have a big issue with that really 🤷